core/src/main/java/fr/sncf/osrd/standalone_sim/SafetySpeed.kt

```diff
@@ -0,0 +1,94 @@
+package fr.sncf.osrd.standalone_sim
+
+import fr.sncf.osrd.api.api_v2.standalone_sim.SimulationScheduleItem
+import fr.sncf.osrd.conflicts.TravelledPath
+import fr.sncf.osrd.railjson.schema.schedule.RJSTrainStop
+import fr.sncf.osrd.sim_infra.api.Path
+import fr.sncf.osrd.sim_infra.api.RawInfra
+import fr.sncf.osrd.sim_infra.api.Route
+import fr.sncf.osrd.sim_infra.api.ZonePath
+import fr.sncf.osrd.sim_infra.api.ZonePathId
+import fr.sncf.osrd.sim_infra.impl.ChunkPath
+import fr.sncf.osrd.utils.DistanceRangeMap
+import fr.sncf.osrd.utils.distanceRangeMapOf
+import fr.sncf.osrd.utils.indexing.StaticIdx
+import fr.sncf.osrd.utils.indexing.StaticIdxList
+import fr.sncf.osrd.utils.units.Offset
+import fr.sncf.osrd.utils.units.Speed
+import fr.sncf.osrd.utils.units.kilometersPerHour
+import fr.sncf.osrd.utils.units.meters
+
+/**
+ * Compute safety speed ranges, areas where the train has a lower speed limit because of a scheduled
+ * stop. For details, see https://osrd.fr/en/docs/reference/design-docs/timetable/#modifiable-fields
+ * (or google "VISA (VItesse Sécuritaire d'Approche)" for resources in French)
```

Comment:
The "google it" part? It's not entirely idea, it doesn't point toward any specific resource (and it's mostly third-party stuff, I didn't find any SNCF document for this).

Ideally we should have a dedicated website page that describes the concept, similar to what we have on the issue with the whiteboard pictures. But this is better than nothing, the method would lack context otherwise

Reply:
Yes, I think that it indeed helps here, and it would help "more" to mention on the website that the French notion for this is named VItesse Securitaire d'Approche (and the result of this web search is currently not so bad I have to say).

Anyway, it's nitpicking, so I resolve it and if I have the opportunity I may add this to the website.